src/main/config/config.h

```diff
@@ -40,11 +40,11 @@ typedef enum {
     FEATURE_RSSI_ADC = 1 << 15,
```

Comment:
Will there be any impact if we clean up the feature bits. (is this used somewhere in the Configurator?)